### PR TITLE
:rocket: Make the gitconfig section more accurate

### PR DESCRIPTION
**Pull request by Rizwanjamal**
:rocket: Make the gitconfig section more accurate

### CODE REVIEW

**Comment by Rizwanjamal** [comment]
@ahmadawais Please review this PR

**Comment by ahmadawais** [comment]
@Rizwanjamal you are not using Emoji log for commits. I cannot in right mind accept this in a repo of Emoji log. Also, I want you to use actual emoji instead of codes like `:rocket:` since that's a GitHub only feature.

File: README.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ funcsave gdoc
 funcsave gtst
 ```
 
-If you prefer, you can paste these aliases directly in your `~/.gitconfig` file:
+If you prefer, you can paste these aliases directly in your `~/.gitconfig` file by using the command `notepad C:/Users/{name}/.gitconfig`, it will open a notepad editor for you containing git configurations.
```
Comment (ahmadawais) [inline README.md:148]: This is a universal repo, description regarding Windows is unnecessary. You can put it all up in an issue and link it there.

File: README.md [hunk 2]
```diff
@@ -154,19 +154,32 @@ If you prefer, you can paste these aliases directly in your `~/.gitconfig` file:
...
   # RELEASE.
-  rlz = "!f() { git cap \"🚀 RELEASE: $@\"; }; f"
+  rlz = "!f() { git cap \"🚀  RELEASE: $@\"; }; f"
   # DOC.
-  doc = "!f() { git cap \"📖 DOC: $@\"; }; f"
+  doc = "!f() { git cap \"📖  DOC: $@\"; }; f"
   # TEST.
-  tst = "!f() { git cap \"✅ TEST: $@\"; }; f"
+  tst = "!f() { git cap \"✅  TEST: $@\"; }; f"
 ```
```
Comment (ahmadawais) [inline README.md:168]: Why added two spaces? It's not what the standard I set here.

File: README.md [hunk 3]
```diff
@@ -154,19 +154,32 @@ If you prefer, you can paste these aliases directly in your `~/.gitconfig` file:
...
+  # Git Commit.
+  cap = "!f() { git commit -m \"$@\"; }; f"
+```
+That's it, now use the commands simply to add wonderful commits using `EMOJI-LOG`, 
+```sh
+git fix 'Fix the scrolling issue'
+[master ec4ff7f] 🐛 FIX: Fix the scrolling issue
+ 1 file changed, 9 insertions(+)
+ create mode 100644 index.js
+```
```
Comment (ahmadawais) [inline README.md:182]: This is all unnecessary.

**Comment by ahmadawais** [comment]
I don't see why any of the changes here are required. So, closing the PR. Doesn't add anything useful but extra Windows related details, you can put all that in an issue and I can link it there.

**Comment by ahmadawais** [comment]
Thanks a lot for taking out time :)

**Pull request closed without merging**